.github/.wordlist.txt

```diff
@@ -1418,6 +1418,9 @@ xFFFF
 xfffff
 xtensa
 xwayland
+xxxxx
+XXXX
+XXXXXXXX
```

Comment:
`/XX+/` are not words and should be not in wordlist. Use backticks in md to make spellcheck happy (and get better format in general)